lib/cc/services/github_pull_requests.rb

```diff
@@ -104,6 +107,9 @@ def update_status(state, description)
         target_url:  @payload["details_url"],
         context:     "codeclimate"
       }
+      if state == "error"
+        params.delete(:target_url)
```

Comment:
@brynary apologies, just saw your note! I believe the thought was that removing the `details_url` would remove the `details` link from PRs that error out.

What's constructed in this `update_status` method doesn't match the original payload perfectly, and without adding in a conditional, it might set the `target_url` to `nil` when `@payload["details_url"]` gets left out.

That said, I think this is the place in `finalizer` where `payload` gets created and sent:
```ruby
  module Finalizer
    class UpdatePullRequest
      def initialize(ref_point)
        @ref_point = ref_point
        @repo = ref_point.repo
        @pull_request = ref_point.pull_request
      end

      def update
        return unless updatable?

        emitter = EventEmitter.new
        emitter.emit(
          "pull_request",
          state: evaluation.state,
          issue_comparison_counts: {
            new: evaluation.new_issue_count,
            fixed: evaluation.fixed_issue_count,
          },
          details_url: evaluation.url,
          compare_url: evaluation.url,
          commit_sha: ref_point.commit_sha,
          github_slug: repo.github_slug,
          repo_id: repo.id.to_s,
          number: pull_request.number,
          message: evaluation.message,
        )
      end
``` 
We could perhaps make a change there.